CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Changed
 
 - AI data is synchronizable between devices via the new functions `syncdata_bytes` and `synchronize`.
+- Add multi-threading support: Parallelism is always enabled for mobile targets and can optionally be enabled for web targets. An optional set of `Feature`s is introduced to pick the assets for the chosen target during the setup (see `getInputData()`), currently the only available `Feature` is `webParallel` (sequential by default). The asset generation and publishing has been updated accordingly.
```

Comment:
Maybe we should note that enabling parallelism for web targets can have problems if the AI isn't completely run in a `Worker`?

Reply:
```suggestion
- Add multi-threading support: Parallelism is always enabled for mobile targets and can optionally be enabled for web targets. An optional set of `Feature`s is introduced to pick the assets for the chosen target during the setup (see `getInputData()`). Currently the only available `Feature` is `webParallel` (sequential by default). The asset generation and publishing has been updated accordingly.
```
also: `(sequential by default)` is a little unclear to me. perhaps it can be removed, or we can add a few extra words e.g. "by default, web assets are loaded sequentially" or whatever.